application_form/migrations/0056_apartmentreservation_list_position.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.2.12 on 2022-04-14 09:27
+
+from django.db import migrations, models
+
+from application_form.models import ApartmentReservation, LotteryEventResult
+
+
+class Migration(migrations.Migration):
+    def migrate_list_position(apps, schema_editor):
+        reservations = ApartmentReservation.objects.all()
+        for reservation in reservations:
+            try:
+                lottery_result = (
+                    reservation.application_apartment.lotteryeventresult.result_position
+                )
+            except LotteryEventResult.DoesNotExist:
+                lottery_result = reservation.queue_position
```

Comment:
This isn't quite enough yet, because `queue_position` can be `null` or non-unique.

Reply:
Fixed!